### PR TITLE
Adding unicycler module

### PR DESCRIPTION
**Pull request by JoseEspinosa**
Add unicycler module as a requirement of nfcore/viralrecon DSL2 implementation as listed [here](https://github.com/nf-core/viralrecon/issues/149)

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [x] If necessary, include test data in your PR.
- [x] Remove all TODO statements.
- [x] Emit the `<SOFTWARE>.version.txt` file.
- [x] Follow the naming conventions.
- [x] Follow the parameters requirements.
- [x] Follow the input/output options guidelines.
- [x] Add a resource `label`
- [x] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
    - [x] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd`
    - [x] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd`
    - [x] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd`

### CODE REVIEW

**Comment by apeltzer** [comment]
Good, good - keep on adding all the modules I need for nf-core/bacass @JoseEspinosa

File: software/unicycler/meta.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+name: unicycler
+description: Assembles bacterial genomes
+keywords:
+  - sort
```
Comment (KevinMenden) [inline software/unicycler/meta.yml:4]: sort? 😛

**Review by KevinMenden — APPROVED** [review]

File: software/unicycler/meta.yml (hunk 1 above)
Comment (KevinMenden) [reply software/unicycler/meta.yml:4]: maybe better keywords and then ready to merge 👍

Comment (JoseEspinosa) [reply software/unicycler/meta.yml]: I left the keyword created by `modules create`... Sort may become a popular `keyword` 😝

**Pull request merged**